src/main/java/com/tqt/javacommon/utils/MessageMapping.java

```diff
@@ -0,0 +1,43 @@
+package com.tqt.javacommon.utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Posted from Jan 01, 2020, 7:09 PM
+ *
+ * @author Vi Quynh (vi.quynh.31598@gmail.com)
+ */
+public class MessageMapping {
+
+    public static Map<Integer, Integer> DVCMappingMICErrCode() {
+        Map<Integer, Integer> messMap = new HashMap<>();
+        messMap.put(MICMessage.SUCCESS.getVal(), DVCMessage.SUCCESS.getVal());
+        messMap.put(MICMessage.FAIL.getVal(), DVCMessage.FAIL.getVal());
+        return messMap;
+    }
+
+    public static Map<Integer, String> DVCMapping() {
+        Map<Integer, String> messMap = new HashMap<>();
+        messMap.put(DVCMessage.SUCCESS.getVal(), DVCMessage.SUCCESS.getMs());
+        messMap.put(DVCMessage.FAIL.getVal(), DVCMessage.FAIL.getMs());
+        return messMap;
+    }
+
+    public static Message mappingCodeProcess(Message message, Integer errCode, String mess) {
```

Comment:
Đây liệu có phải là một common khi chúng ta truyền giá trị và  errCode thì nó sẽ tìm kiếm và set 1 số thuộc tính khác cho chúng ta hay không ?
Là một common hay dùng nên ghi rõ tác dụng và cách sử dụng cơ bản.

Reply:
@solitarysp Có tách được phần for if, for if kia ra khoong?